nuget.config

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+ <packageSources>
+    <!--To inherit the global NuGet package sources remove the <clear/> line below -->
+    <clear />
```

Comment:
@eerhardt will this affect anything else, or will they all keep using the MSBuild source?

Reply:
I'd like to not have this list duplicated. NuGet.configs are not great because they cannot be modified (either adding new sources or removing sources, or changing locations) without actually modifying the file.

Instead, would it be possible to:

1. pull the `RestoreSources` property out of `Directory.Build.props` and put it in a separate file, say `build\NuGetSources.props`. `Directory.Build.props` imports that file.
2. Change the Benchmark toolchain to import the `build\NuGetSources.props` file.

That way we don't need to duplicate this list.

---
In reply to: [220023911](https://github.com/dotnet/machinelearning/pull/1001#discussion_r220023911) [](ancestors = 220023911)